### ISSUE TITLE
Color splashscreen at boot when espanso is launched (also occurs at restart)

### ISSUE THREAD

**Issue by nnuel** [federico-terzi/espanso#972]
**Describe the bug**
It shows a rainbow colorwheel splashscreen when starting espanso. This happens right after boot, but can reproduce when restarting the program.

**To Reproduce**
1. `espanso restart`

**Expected behavior**
To have it not open and close a window. Just launch in the background. 

**Screenshots**
I can't really record this in a screenshot as the flash is really quick, I could make a video, but maybe it is clear what happens. 

**Logs**
![image](https://user-images.githubusercontent.com/4148154/151990554-e3be99b7-a6ed-4e27-a7cc-4d829996344e.png)

**Setup information**
- OS: Arch/Sway
- Version: 2.1.2-alpha

**Comment by federico-terzi** [comment]
Hey @nnuel,

I'm sorry you experienced this problem! Unfortunately, this is not something we can work around on wayland at the moment :(
Espanso needs to spawn a window to gather the current keyboard information, otherwise it goes out of sync.
I know it's slightly inconvenient, but I couldn't find any other way to solve the problem while keeping Espanso in the background...

If anyone finds another way to solve the problem, feel free to reopen the issue to discuss this further! For reference, this is the relevant code: https://github.com/federico-terzi/espanso/blob/master/espanso-detect/src/evdev/sync/wayland.rs

Hope you'll understand :)

Cheers :)

**Comment by nnuel** [comment]
Ah, no problem, as long as it is necessary for the program to function, then it clearly outweighs the minor nuisance. Can the type of window / screen be determined. Now it has rainbow colors and I am wondering if it could work with a simple black screen. Maybe because I am using a tiling window manager that it takes up the fullscreen or is this the same for other normal DEs too?

**Comment by federico-terzi** [comment]
@nnuel Yes, the window manager might make it more evident (usually the window should be small). I've opened an issue with the improvements you suggested :) https://github.com/federico-terzi/espanso/issues/976
